src/main/java/uk/gov/hmcts/ccd/domain/service/validate/CaseDataIssueLogger.java

```diff
@@ -0,0 +1,80 @@
+package uk.gov.hmcts.ccd.domain.service.validate;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.collections.MapUtils;
+import uk.gov.hmcts.ccd.domain.model.definition.CaseDetails;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@Slf4j
+public class CaseDataIssueLogger {
+
+    private static final String EMPTY_COLLECTION_VALUE = "{}";
+
+    private CaseDataIssueLogger() {
+    }
+
+    public static List<String> logAnyDataIssuesIn(CaseDetails currentCaseDetails, CaseDetails newCaseDetails) {
+        try {
+            return logAnyUnexpectedEmptyCollectionIssueIn(currentCaseDetails, newCaseDetails);
```

Comment:
Could we please make this conditional only for a configurable list of jurisdictions? The application property can be defaulted to only the suffering jurisdiction.

Reply:
Completed